catalogs/catalog-jdbc-doris/src/main/java/org/apache/gravitino/catalog/doris/DorisTablePropertiesMetadata.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.catalog.doris;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Maps;
+import java.util.List;
+import java.util.Map;
+import org.apache.gravitino.connector.BasePropertiesMetadata;
+import org.apache.gravitino.connector.PropertyEntry;
+
+public class DorisTablePropertiesMetadata extends BasePropertiesMetadata {
+
+  public static final String REPLICATION_FACTOR = "replication_allocation";
+  public static final String DEFAULT_REPLICATION_FACTOR = "tag.location.default: 1";
+
+  private static final Map<String, PropertyEntry<?>> PROPERTIES_METADATA;
+
+  static {
+    List<PropertyEntry<?>> propertyEntries =
+        ImmutableList.of(
+            // TODO: make REPLICATION_FACTOR mutable if needed
+            PropertyEntry.stringOptionalPropertyEntry(
+                REPLICATION_FACTOR,
+                "The number of replications for the table, if not specified, ",
+                true /* immutable */,
```

Comment:
Why make it immutable?

Reply:
I'm hesitate to make it mutable though it's indeed mutable in Doris table as I not aware whether we need to change it in production environment.

Reply:
I think we should follow the engine's way. 

Can you help confirm if there is a need to change the parameter in production? @xiaozcy @zhoukangcn

Reply:
Doris allows modification of the number of replications, I think we'd better make it mutable.

Reply:
OK